packages/framework-core/src/services/read-model-store.ts

```diff
@@ -69,7 +73,7 @@ export class ReadModelStore {
     return joinKey
   }
 
-  private reducerForProjection(projectionMetadata: ProjectionMetadata): Function {
+  public reducerForProjection(projectionMetadata: ProjectionMetadata): Function {
```

Comment:
Why is this public now?

Reply:
Just for testing purposes @NickSeagull: 
```
replace(
          ReadModelStore.prototype,
          'reducerForProjection',
          fake.returns(() => deleteReadModel)
        )
```

Reply:
You can use the `rewire` library to load private functions without exposing them. There's an example here: https://github.com/boostercloud/booster/blob/master/packages/cli/test/commands/deploy.test.ts#L11

Reply:
Hey @javiertoledo I've been trying to do that but I can't find the way of "stubbing" exported classes methods. All examples I found are from functions outside classes. I can get the ReadModelStore and even print it but it's printed as a function, and I think that's a rewire limitation: https://www.npmjs.com/package/rewire#limitations.

```
const readModelStoreRewire = rewire('../../src/services/read-model-store')
        readModelStoreRewire.__set__({
          ReadModelStore: {
            config,
            logger,
            reducerForProjection: () => null,
          },
        })
        const readModelStore = new readModelStoreRewire.ReadModelStore(config, logger)

        await readModelStore.project(anEntitySnapshot)
```

⬆️ The `project` method gets called but ignores my setters